chef/cookbooks/provisioner/templates/default/autoyast.xml.erb

```diff
@@ -96,8 +104,38 @@
           <device>/dev/<%= @boot_device %></device>
         <% end %>
         <initialize config:type="boolean">true</initialize>
-        <partitions config:type="list"/>
         <type config:type="symbol">CT_DISK</type>
+        <disklabel>gpt</disklabel>
```

Comment:
For larger disks, we need to specify gpt disklabel. I am not so sure if we break the non-GPT setups ( AFAIK, it should work with GPT. Not sure why this breaks non-GPT setups?).